src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -398,7 +398,8 @@ private boolean moveRubricCol(int qnNumber, int colNumber, boolean isMoveLeft) {
         WebElement moveColButton = browser.driver.findElement(By.id(elemId));
 
         if (moveColButton.getAttribute("disabled") == null) {
-            moveColButton.click();
+            waitForElementToBeClickable(moveColButton);
```

Comment:
In my opinion, It's always better to add a wait before the clicks, I think it improves the success rate of the tests. It's also there before most clicks in the tests haha. 

@whipermr5 Is this also defensive coding? And would you recommend keeping it or removing it?